server/src/main/java/io/druid/client/HttpServerInventoryView.java

```diff
@@ -96,6 +96,11 @@
 
   private final ConcurrentMap<SegmentCallback, Predicate<Pair<DruidServerMetadata, DataSegment>>> segmentPredicates =
       new ConcurrentHashMap<>();
+
+  // Users of this instance can register filters for what segments should be stored and reported to registered
```

Comment:
Could you please make this a javadoc comment?

Reply:
you mean on the constructor ?

Reply:
on the field

Reply:
just curious, why does that matter ?

Reply:
- More apparent that this comment belongs to a field, not just random comment between the lines
 - When code is refactored in IDE, the doc comment is attached to a field and moves with it, unlike `//` comments
 - Ability to make other class refs in a comment links, to ease navigation in IDE, and also when classes are renamed, links in Javadocs are renamed automatically too.

Reply:
ok, changed.